File: src/main/kotlin/net/atos/zac/zoeken/model/zoekobject/ZoekObject.kt
/*
 * SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.zoeken.model.zoekobject

interface ZoekObject {
    companion object {
        const val IS_TOEGEKEND_FIELD: String = "isToegekend"
    }

    fun getId(): String

    fun getType(): ZoekObjectType
}

File: src/main/app/src/app/zoeken/model/zoek-object.ts
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */

import { GeneratedType } from "../../shared/utils/generated-types";

/**
 * @deprecated - use the `GeneratedType`
 */
export interface ZoekObject {
  id: string;
  type: GeneratedType<"ZoekObjectType">;
}
